presto-product-tests/src/main/java/io/prestosql/tests/hive/TestSyncPartitionMetadata.java

```diff
@@ -152,6 +155,25 @@ private static void prepare(HdfsClient hdfsClient, HdfsDataSourceWriter hdfsData
         assertPartitions(tableName, row("a", "1"), row("b", "2"));
     }
 
+    private static String getTableLocation(String tableName, int partitionLevels)
```

Comment:
why not get location from `show create table` output?

Reply:
Presto doesn't show that for managed tables and i didn't want to go to hive for information that Presto has.

(fyi we use this apprach in couple other tests already)